src/features/SensorStatus/SensorStatusManager.ts

```diff
@@ -63,14 +83,27 @@ export class SensorStatusManager {
     this.databaseService = databaseService;
   }
 
+  mapStatusQueryResult = (result: SensorStatusQueryResult): SensorStatus => {
```

Comment:
trying to be more explicit about the typing and safer with the values returned from the raw query